spack.yaml

```
@@ -6,17 +6,14 @@
 # https://github.com/ACCESS-NRI/model-deployment-template/blob/main/spack.yaml
 spack:
   specs:
-    - access-om3@git.2024.09.0
+    - access-om3@git.2024.12.0
   packages:
-    # Main Dependencies
     access-om3-nuopc:
       require:
-        - '@git.0.3.1'
-
-    # Other Dependencies
+        - '@git.209-new-build'
```

```suggestion
        - +mom_symmetric
```

See https://github.com/COSIMA/access-om3/issues/255